ts/screens/onboarding/EmailValidateScreen.tsx

```diff
@@ -0,0 +1,134 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { untag } from "italia-ts-commons/lib/types";
+import { Button, Text, View } from "native-base";
+import * as React from "react";
+import { Alert, StyleSheet } from "react-native";
+import { NavigationScreenProps } from "react-navigation";
+import { connect } from "react-redux";
+
+import ScreenContent from "../../components/screens/ScreenContent";
+import TopScreenComponent from "../../components/screens/TopScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import Markdown from "../../components/ui/Markdown";
+import I18n from "../../i18n";
+import { BiometrySimpleType } from "../../sagas/startup/checkAcknowledgedFingerprintSaga";
+import {
+  abortOnboarding,
+  emailAcknowledged
+} from "../../store/actions/onboarding";
+import { Dispatch, ReduxProps } from "../../store/actions/types";
+import { GlobalState } from "../../store/reducers/types";
+
+type NavigationParams = {
+  biometryType: BiometrySimpleType;
+};
+
+type OwnProps = ReduxProps &
+  ReturnType<typeof mapStateToProps> &
+  NavigationScreenProps<NavigationParams>;
+
+type Props = OwnProps & ReturnType<typeof mapDispatchToProps>;
+
+const styles = StyleSheet.create({
+  spacerLarge: { height: 40 }
+});
+
+const unavailableAlert = () => Alert.alert(I18n.t("global.notImplemented"));
+
+/**
+ * A screen to show if the fingerprint is supported to the user.
+ */
+export class FingerprintScreen extends React.PureComponent<Props> {
+  private handleGoBack = () =>
+    Alert.alert(
+      I18n.t("onboarding.alert.title"),
+      I18n.t("onboarding.alert.description"),
+      [
+        {
+          text: I18n.t("global.buttons.cancel"),
+          style: "cancel"
+        },
+        {
+          text: I18n.t("global.buttons.exit"),
+          style: "default",
+          onPress: () => this.props.abortOnboarding()
+        }
+      ]
+    );
+
+  public render() {
+    const { potProfile } = this.props;
+
+    const profileData = potProfile
```

Comment:
Could we have email directly?= https://github.com/teamdigitale/io-app/pull/1308#discussion_r327027072

Reply:
changed in cdb554f8